src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = {
```

Comment:
It's good that you've created an instance of FigureSupplier before the loop. However, you should consider creating more than three figures for a more robust example. You can use a loop to create a dynamic number of figures.